### DISCUSSION TITLE
Not showing or kot download option in Release section.

### DISCUSSION THREAD

**Discussion by exatonix** [community/community#144357] [Mobile]
When in chrome we can see release section and can be downloadable release files apk jpg exe deb  etc.

but in mobile github app
only showing us release version history and datecodes etc. not showing release files to show see or download.

please fix it as soon as.

**Comment by HenSquared** [comment]
Hi @exatonix , thanks for the feedback. I've added an internal issue for us to track this ask.

**Reply by exatonix** [reply]
thanks. it is worked now.I senr here for feedback latest Github Bug Fixed.

![Uploading Screenshot_2024-12-02-19-49-45-509_com.android.chrome.jpg…]()

**Comment by exatonix** [comment]
still same problem in new version

**Reply by exatonix** [reply]
@HenSquared still same problem in some pages. I think maybe about Xiaomi devices and maybe about Android WebView of google internal problem

**Comment by jacobjohnsons1** [comment]
That’s definitely something that needs addressing—having access to release files on mobile is super important for quick testing [Gold Rate](https://mypricezone.com/) or downloads on the go.
